okfn/opendataeditor
# Remove example project

fixes #328 

I'm also renaming `APP_EXAMPLE` to `APP_TMP` to make more explicit that it is only a temporary folder.

## How to test

1. Build the application
2. Install it
3. Open the application (by executing the application, not by running `make start`)
  a. Note: `make start` points the project folder of the application to another folder names `./tmp/project`
5. Clean or backup your `/home/<user>/.opendataeditor` folder.
6. On first time run there shouldn't be any example files loaded.
7. The default project should point to `/home/<user>/.opendataeditor/tmp`

![image](https://github.com/okfn/opendataeditor/assets/6672339/94cb7fb1-b87c-448f-b851-f5a161098515)
